app/src/main/java/com/hover/stax/data/repository/BountyRepositoryImpl.kt

```diff
@@ -20,16 +18,6 @@ class BountyRepositoryImpl(val actionRepo: ActionRepo, private val coroutineDisp
     override val bountyActions: List<HoverAction>
         get() = actionRepo.bounties
 
-    override fun isSimPresent(bounty: Bounty, sims: List<SimInfo>): Boolean {
```

Comment:
This is duplication of a method that is elsewhere. It doesn't belong here, why would bonus repo have it's own version of this?